### PR TITLE
feat: switch deprecated cf-ipfs link to dweb

### LINKED ISSUE #25: Change cf-ipfs link to dweb
cf-ipfs is deprecated in favor of dweb now https://blog.cloudflare.com/cloudflares-public-ipfs-gateways-and-supporting-interplanetary-shipyard

### PR DESCRIPTION
**Pull request by samsiegart**
fixes https://github.com/Agoric/dapp-inter-jumper/issues/25

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ const getRedirectUrl = async (apiAddrs: string[]) => {
   const body = JSON.parse(value.body.substring(1));
   // TODO: remove backwards-compatible "EndorsedUI" after contract change.
   const referencedUI = body.current.ReferencedUI || body.current.EndorsedUI;
```
Comment (turadg) [inline src/main.ts]: drive-by: can we do this TODO now?

**Review by turadg — APPROVED** [review]
c

File: src/main.ts (hunk 1 above)
Comment (samsiegart) [reply src/main.ts]: Yea seems to work with mainnet, devnet, and emerynet, so removed this now.
